wagtail/images/templates/wagtailimages/images/view_toggle_button.html

```diff
@@ -0,0 +1,28 @@
+{% load wagtailadmin_tags i18n %}
+
+<div>
+    {% if view_mode == "list" %}
+        <a href="?view=grid">
+            <button
+                type="button"
+                class="w-filter-button"
+                aria-label="{% trans 'Toggle view' %}"
```

Comment:
This should be switched to a tooltip like we have for the existing filter button. And changed the label to `Toggle layout`.